lib/uri.mli

```diff
@@ -252,5 +252,10 @@ val with_fragment : t -> string option -> t
 
 (** {3 Utilities } *)
 
-(**  Human-readable output, used by the toplevel printer *)
-val pp_hum : Format.formatter -> t -> unit
+(**  [pp ppf t] will output a human readable version of the Uri [t]
+    to the formatter [ppf] *)
+val pp : Format.formatter -> t -> unit [@@ocaml.toplevel_printer]
+
+(**  [pp_hum] is now an alias for the {!pp} function. *)
+val pp_hum : Format.formatter -> t -> unit [@@ocaml.toplevel_printer]
```

Comment:
How does it work when there are several tagged printers like in this case?

Reply:
Isn't the same way as other toplevel printers? The full type has to be an exact string match if I understand it correctly (so another `Format.formatter -> Uri.t -> unit` would overwrite this one). cc @diml to make sure I've used it right here.

Reply:
I meant that in this mli `pp_hum` and `pp` are both tagged for the same type. Is this intended?

Reply:
oh, no! good catch :)

Reply:
i was going to deprecate `pp_hum` too, but could save that for a future release...

Reply:
fixed in 4778cf41a7d74a7118f74e21c8ece141595f4c50